es6.md

```diff
@@ -189,3 +189,35 @@ interface ImportNamespaceSpecifier {
 ```
 
 A namespace import specifier, e.g., `* as foo` in `import * as foo from "mod.js"`.
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Node {
```

Comment:
Aren't we trying to flesh out a structure that makes the most sense here? Seems it'd be a shame if we got this in only to decide later to come back and make a second breaking change for various places.